### ISSUE TITLE
bug: `EstimatorReport` helper display in rich

### ISSUE THREAD

**Issue by sylvaincom** [probabl-ai/skore#1180]
### Describe the bug

The return to line at the level of `.cache_predictions(...)` is badly handled:

<img src="https://github.com/user-attachments/assets/b11c9956-bcd0-4f78-8b05-0b4127b9a24e" width="600">

### Steps/Code to Reproduce

```python
from sklearn.linear_model import LinearRegression
from sklearn.datasets import make_regression
from skore import CrossValidationReport

data, target = make_regression(random_state=42)
regressor = LinearRegression()
reporter = CrossValidationReport(regressor, data, target)
reporter.help()
```

### Expected Behavior

The return to line should stay on the right (right column)

### Actual Behavior

The return to line should not go on the left column

### Environment

```shell
System:
    python: 3.12.4 | packaged by Anaconda, Inc. | (main, Jun 18 2024, 10:07:17) [Clang 14.0.6 ]
executable: /Users/sylvaincom/Documents/GitHub/internal_demos/.venv/bin/python
   machine: macOS-15.1.1-arm64-arm-64bit

Python dependencies:
        skore: 0.6.0rc1
          pip: 24.3.1
   setuptools: None
    diskcache: 5.6.3
      fastapi: 0.115.6
       joblib: 1.4.2
   matplotlib: 3.9.3
        numpy: 2.2.0
       pandas: 2.2.3
       plotly: 5.24.1
      pyarrow: 18.1.0
         rich: 13.9.4
 scikit-learn: 1.6.0
        skops: 0.11.0
      uvicorn: 0.32.1
```

**Comment by auguste-probabl** [comment]
The console is already at 88 chars

**Comment by glemaitre** [comment]
So we need larger. I don't no what is best.

**Comment by MarieS-WiMLDS** [comment]
This issue can be closed once we switch from rich to html (as planned in this issue https://github.com/probabl-ai/skore/issues/1104).  
rich back-end will be kept for code executed in a terminal. In that use case it is clear that we can't keep extend the console window.

**Comment by glemaitre** [comment]
Uhm it does not seem trivial because you need to detect the padding and understand when the line break happen and it seems to be a `rich` thingy. But maybe it is feasible.

**Comment by glemaitre** [comment]
I would say that we cannot do much more than what `rich` propose and it will be a best effort. I have more hope when it will come using the HTML representation to handle beautifully those case.
